extensionsGallery.json

```diff
@@ -4126,7 +4126,7 @@
 								},
 								{
 									"key": "Microsoft.AzDataEngine",
-									"value": ">=1.33.0"
+									"value": ">=1.35.0"
```

Comment:
FYI - it doesn't matter a whole lot but you should be keeping the engine version in your package.json in line with this : https://github.com/microsoft/azuredatastudio/blob/main/extensions/sql-migration/package.json#L13

I'd suggest updating it there now - but for this release we can probably just skip that (just be careful in the future that whenever you make changes to the extension that requires an updated ADS version you should be updating this as well)

Reply:
PR for updating package.json https://github.com/microsoft/azuredatastudio/pull/18462.
Thank You @Charles-Gagnon for keeping track of changes.